Bartender/Bartender.jss.recipe

```diff
@@ -17,7 +17,7 @@
 		<key>NAME</key>
 		<string>Bartender</string>
 		<key>OS_REQUIREMENTS</key>
-		<string>10.13.x,10.12.x,10.11.x,10.10.x</string>
+		<string>10.15.x,10.14.x,10.13.x,10.12.x,10.11.x,10.10.x</string>
```

Comment:
According to the Bartender website, the latest version requires 10.12+.
<img width="412" alt="image" src="https://user-images.githubusercontent.com/7801391/71314390-44ba1d00-23fc-11ea-9129-9ab29051dfbc.png">